src/ovirtapi/transform.js

```diff
@@ -478,14 +478,15 @@ const Snapshot = {
       type: snapshot.snapshot_type || '',
       date: snapshot.date || Date.now(),
       status: snapshot.snapshot_status || '',
-      persistMemoryState: snapshot.persist_memorystate === 'true',
+      persistMemoryState: convertBool(snapshot.persist_memorystate),
       isActive: snapshot.snapshot_type === 'active',
     }
   },
 
   toApi ({ snapshot }: { snapshot: SnapshotType }): ApiSnapshotType {
     return {
       description: snapshot.description,
+      persist_memorystate: Boolean(snapshot.persistMemoryState).toString(),
```

Comment:
Go ahead and convert over to a `type ApiBooleanType = "true" | "false"` and use a function like:

```js
function toApiBoolean (value: any): ApiBooleanType {
  return Boolean(value).toString()
}
```

to handle the transform.  A followup PR, like @sgratch suggested, to have the API boolean type applied across all of the types and transforms.

Reply:
I decided to implement it differently because I got a flow error:
![image](https://user-images.githubusercontent.com/64131213/115143454-0f552c00-a050-11eb-8ed5-9be218f995e8.png)
please let me know WDYT :)